lib/beemutex.cpp

```diff
@@ -32,8 +32,8 @@
 
 #include "common.h"
 
-#include <errno.h>
-#include <string.h>
+#include <cerrno>
+#include <cstring>
```

Comment:
This sort of thing has zero practical benefit and at least one downside, being that there are compilers that don't provide these alternate files, so it breaks compatibility.